src/views/inspector/InspectorViewProvider.ts

```diff
@@ -27,17 +28,21 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
     if (!folder) {
       return
     }
-    this.variablesCLIController = new VariablesCLIController(folder)
-    this.featuresCLIController = new FeaturesCLIController(folder)
 
-    this.variablesCLIController.getAllVariables().then((variables) => {
+    this.matches = StateManager.getFolderState(folder.name, KEYS.CODE_USAGE_KEYS) || {}
+    
+    const variablesCLIController = new VariablesCLIController(folder)
+    const featuresCLIController = new FeaturesCLIController(folder)
+
+    variablesCLIController.getAllVariables().then((variables) => {
       this.variables = variables
+      // default to the alphabetically first variable in variable list
       if (Object.values(this.variables).length) {
-        this.selectedKey = Object.values(this.variables)[0].key
+        this.selectedKey = Object.values(this.variables).sort((a: Variable, b: Variable) => a.name.localeCompare(b.name))[0].key
```

Comment:
we should probably sort the variables once, and use the ordered result both here and to render the options below